portal/app/components/blacklight_range_limit/range_facet_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= render(@layout.new(facet_field: @facet_field)) do |component| %>
```

Comment:
This file was copied with no changes in order for range_facet_component.rb to work.